### PR TITLE
doc(checkout): CHECKOUT-8739 update doc to reflect new feature of order level fees.

### PR DESCRIPTION
**Pull request by bc-shawnwang**
# [CHECKOUT-8739](https://bigcommercecloud.atlassian.net/browse/CHECKOUT-8739)

## What changed?
v3 checkout api:
- ~~Create Fee~~ (see "Updates after change to bulk APIs")
- ~~Update Fee~~ (see "Updates after change to bulk APIs")
- ~~Delete Fee~~ (see "Updates after change to bulk APIs")
- response of checkout object added applied fees (fees:)

storefront checkout api:
- response of checkout object added applied fees (fees:)

-----

<details open>
<summary> <h3> Updates after change to bulk APIs, 2024-11-28 </h3> </summary>

on commit: https://github.com/bigcommerce/docs/pull/642/commits/5dd480c9c0107b6b6df0c5e735630fb2376222b9

v3 checkout api changes:

Have 3 endpoints to support bulk operations on fees:
- POST v3/checkouts/:checkoutId/fees
- PUT v3/checkouts/:checkoutId/fees
- DELETE v3/checkouts/:checkoutId/fees

Removed 2 endpoints:
- PUT v3/checkouts/:checkoutId/fees/:feeId
- DELETE v3/checkouts/:checkoutId/fees/:feeId

Response of v3 checkout and storefront checkout have nothing changed.

### Overall 
image

### bulk POST
image

### bulk PUT
Screenshot 2024-11-28 at 10 11 24 AM

### bulk DELETE
image

</details>

## Release notes draft

* The newly-released custom order level fees on the checkout is now available to use. Now, you’ll be able to apply and manage the checkout's order level fee via v3 management api, and fetch any checkout's applied fees on storefront checkout api and v3 management checkout api. 

## Anything else?

ping @bigcommerce/team-checkout @bigcommerce/dev-docs 

[CHECKOUT-8739]: https://bigcommercecloud.atlassian.net/browse/CHECKOUT-8739?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by bc-shawnwang** [comment]
@bigcommerce/dev-docs we do not merge it until GA of the feature, but open the PR for review ahead of time.

**Review by valentindellangela — DISMISSED** [review]

File: reference/checkouts.sf.yml
Comment (valentindellangela) [inline reference/checkouts.sf.yml:1179]: LGTM, I'm just not sure if we should be using Route as example or give a random one instead to avoid the advertisment 😄

File: reference/checkouts.sf.yml
Comment (bc-traciporter) [inline reference/checkouts.sf.yml:1174]:
```suggestion
          description: Cost of the fee (include or exclude tax dependent on tax settings, same as shipping cost).
```

File: reference/checkouts.sf.yml
Comment (bc-traciporter) [inline reference/checkouts.sf.yml:1178]:
```suggestion
          description: The source of the request.
```

File: reference/checkouts.sf.yml
Comment (bc-traciporter) [inline reference/checkouts.sf.yml:1182]:
```suggestion
          description: The tax class ID.
```

File: reference/checkouts.v3.yml
Comment (bc-traciporter) [inline reference/checkouts.v3.yml:7777]:
```suggestion
          - Maximum of 5 fees per checkout.
```

File: reference/checkouts.v3.yml
Comment (bc-traciporter) [inline reference/checkouts.v3.yml:8890]:
```suggestion
          description: The type of the fee.
```

File: reference/checkouts.v3.yml
Comment (bc-traciporter) [inline reference/checkouts.v3.yml:8899]:
```suggestion
          description: Display name of the fee targeting customers/shoppers.
```

File: reference/checkouts.v3.yml
Comment (bc-traciporter) [inline reference/checkouts.v3.yml:8903]:
```suggestion
          description: Cost of the fee including tax.
```

File: reference/checkouts.v3.yml
Comment (bc-traciporter) [inline reference/checkouts.v3.yml:8907]:
```suggestion
          description: Cost of the fee excluding tax.
```

File: reference/checkouts.v3.yml
Comment (bc-traciporter) [inline reference/checkouts.v3.yml:8911]:
```suggestion
          description: The source of the request.
```

File: reference/checkouts.v3.yml
Comment (bc-traciporter) [inline reference/checkouts.v3.yml:8916]:
```suggestion
          description: The tax class ID.
```

File: reference/checkouts.v3.yml
Comment (bc-traciporter) [inline reference/checkouts.v3.yml:9238]:
```suggestion
          description: The tax class ID applied to this fee (the tax class ID can be retrieved from our management API - v2/tax_classes). If the tax class is not provided or is null, the default fee tax class set in the control panel will be used.
```

**Review by bc-traciporter — COMMENTED** [review]
Looks fine. I made some editorial suggestions to try to keep descriptions consistent.

File: reference/checkouts.sf.yml
Comment (bc-shawnwang) [reply reference/checkouts.sf.yml:1179]: yes, good point. let me change it lol

**Review by bc-traciporter — APPROVED** [review]
Approved

File: reference/checkouts.v3.yml
Comment (bc-traciporter) [inline reference/checkouts.v3.yml:9277]:
```suggestion
          description: The IDs of the fees to be deleted from a checkout.
```

File: reference/checkouts.v3.yml
Comment (bc-traciporter) [inline reference/checkouts.v3.yml:9251]:
```suggestion
          description: The fees to be added to a checkout.
```

File: reference/checkouts.v3.yml
Comment (bc-traciporter) [inline reference/checkouts.v3.yml:9239]:
```suggestion
              description: ID of the fee.
```

File: reference/checkouts.v3.yml
Comment (bc-traciporter) [inline reference/checkouts.v3.yml:7814]:
```suggestion
        > We do not support partial updates, so please send the total entity values for each fee to be updated.
```

File: reference/checkouts.sf.yml
Comment (bc-traciporter) [inline reference/checkouts.sf.yml:978]:
```suggestion
          description: Fees applied at the checkout level.
```

**Review by bc-traciporter — COMMENTED** [review]
I made some suggestions.

**Review by bc-traciporter — APPROVED** [review]
Approved

**Pull request merged**
